app/policies/mixins/write_policy_mixin.rb

```diff
@@ -1,9 +1,12 @@
 module WritePolicyMixin
   def create?
+    admin? || system?
```

Comment:
if I understand well, `admin?` can do API request to RBAC service - so in order to avoid some requests - is it beneficial to do `system?` first ? 

(Not sure whether order is intended)

Reply:
So I couldn't remember so I went back and tried again - turns out there was a spec failure and that failure was the fact that it blew up when the x-rh-identity wasn't there. I think I flipped the order initially due to the fact I hadn't disabled RBAC globally yet.

I added a rescue on the `IdentityError` exception that gets thrown and returned false - since it obviously isn't system-authorized if there isn't a x-rh-identity header.